### ISSUE TITLE
Certain inputs play a single long tone before triggering an error

### ISSUE THREAD

**Issue by Garrett-Bodley** [Garrett-Bodley/SHAlala#11]
input: `ijfqijqf`
corresponding hash: `ae01eb812378013e684f4797417f009c05943417`

console output:
```
Debug.ts:10 Uncaught (in promise) Error: Start time must be strictly greater than previous start time
    at Bn (Debug.ts:10:9)
    at sr.start (Source.ts:207:4)
    at kr._triggerEnvelopeAttack (Synth.ts:109:19)
    at kr.triggerAttack (Monophonic.ts:74:8)
    at kr.triggerAttackRelease (Instrument.ts:156:8)
    at play (index.js:220:17)
    at HTMLFormElement.handleOnSubmit (index.js:223:3)
```

**Comment by gregsadetsky** [comment]
Ah yeah, that will happen if the time value is 0, sorry! There should be a floor so that the time is always greater than 0

**Comment by gregsadetsky** [comment]
fixed in 19dbab6f9c3012d1919c794954b4eaeb58956444
